src/mame/drivers/dvk_ksm.cpp

```diff
@@ -206,15 +209,28 @@ static INPUT_PORTS_START( ksm )
 	PORT_DIPSETTING(0x0E, "75")
 INPUT_PORTS_END
 
+void ksm_state::device_timer(emu_timer &timer, device_timer_id id, int param, void *ptr)
```

Comment:
This is probably a better application for the `clock_device` than a device timer.  Then you can connect its output to the 8251 TxC/RxC by chaining writeline callbacks in MCFG and avoid the extra code altogether.